Uncaught Error: Cannot find module './context-menu

I right-clicked on a folder in the tree view

**Atom Version**: 0.194.0
**System**: Windows 7 Entreprise
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Cannot find module './context-menu'
Error: Cannot find module './context-menu'
    at Function.Module._resolveFilename (module.js:328:15)
    at Function.Module._load (module.js:270:25)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at BrowserWindow.<anonymous> (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\app.asar\src\browser\atom-window.js:152:27)
    at emitOne (events.js:77:13)
    at BrowserWindow.emit (events.js:166:7)
    at callFunction (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\atom.asar\browser\lib\rpc-server.js:116:18)
    at EventEmitter.<anonymous> (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\atom.asar\browser\lib\rpc-server.js:208:14)
    at emitMany (events.js:108:13)

```
At C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\atom.asar\renderer\api\lib\remote.js:77

Error: Cannot find module './context-menu'
Error: Cannot find module './context-menu'
    at Function.Module._resolveFilename (module.js:328:15)
    at Function.Module._load (module.js:270:25)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at BrowserWindow.<anonymous> (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\app.asar\src\browser\atom-window.js:152:27)
    at emitOne (events.js:77:13)
    at BrowserWindow.emit (events.js:166:7)
    at callFunction (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\atom.asar\browser\lib\rpc-server.js:116:18)
    at EventEmitter.<anonymous> (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\atom.asar\browser\lib\rpc-server.js:208:14)
    at emitMany (events.js:108:13)
  at metaToValue (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\atom.asar\renderer\api\lib\remote.js:77:15)
  at BrowserWindow.RemoteMemberFunction [as emit] (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\atom.asar\renderer\api\lib\remote.js:111:26)
  at ContextMenuManager.module.exports.ContextMenuManager.showForEvent (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\app.asar\src\context-menu-manager.js:170:31)
  at HTMLDocument.<anonymous> (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\app.asar\src\window-event-handler.js:149:33)
  at HTMLDocument.handler (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\app.asar\src\space-pen-extensions.js:112:34)
  at HTMLDocument.jQuery.event.dispatch (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\app.asar\node_modules\space-pen\vendor\jquery.js:4681:9)
  at HTMLDocument.elemData.handle (C:\Users\jbrichardet\AppData\Local\atom\app-0.194.0\resources\app.asar\node_modules\space-pen\vendor\jquery.js:4359:46)

```

### Commands

```
     -4:55.5.0 editor:checkout-head-revision (atom-text-editor.editor.is-focused)
  2x -3:41.6.0 window:focus-pane-on-right (atom-text-editor.editor)
     -3:17.5.0 application:add-project-folder (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -2:47.4.0 editor:newline (atom-text-editor.editor.is-focused)
     -2:38.2.0 core:cut (atom-text-editor.editor)
     -2:36.5.0 core:paste (atom-text-editor.editor.is-focused)
     -2:26.6.0 core:save (atom-text-editor.editor.is-focused)
     -2:20.6.0 core:move-down (atom-text-editor.editor)
     -2:20.4.0 autocomplete-plus:confirm (atom-text-editor.editor)
     -2:15.8.0 core:save (atom-text-editor.editor)
     -2:08.7.0 core:copy (atom-text-editor.editor.is-focused)
     -2:01.2.0 core:paste (atom-text-editor.editor.is-focused)
     -1:59.7.0 core:save (atom-text-editor.editor.is-focused)
     -1:52.2.0 core:paste (atom-text-editor.editor.is-focused)
     -1:51.6.0 core:save (atom-text-editor.editor.is-focused)
     -1:30.6.0 core:backspace (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      "node_modules"
    ],
    "themes": [
      "atom-dark-ui",
      "seti-syntax"
    ],
    "disabledPackages": [
      "Tern"
    ],
    "projectHome": "Y:\\app-tfoumax"
  },
  "editor": {
    "invisibles": {},
    "softWrap": true,
    "showIndentGuide": true
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v2.12.0
autocomplete-snippets, v1.2.0
javascript-snippets, v1.0.0
jshint, v1.3.5
language-ejs, v0.1.0
linter, v0.12.1
pretty-json, v0.3.3
save-session, v0.14.0
Search, v0.4.0
seti-syntax, v0.4.0

# Dev
No dev packages
```

--- a/src/browser/atom-application.coffee
+++ b/src/browser/atom-application.coffee
@@ -273,9 +273,8 @@ class AtomApplication
     ipc.on 'cancel-window-close', =>
       @quitting = false
 
-    clipboard = null
+    clipboard = require '../safe-clipboard'
     ipc.on 'write-text-to-selection-clipboard', (event, selectedText) ->
-      clipboard ?= require '../safe-clipboard'
       clipboard.writeText(selectedText, 'selection')
 
   # Public: Executes the given command.

--- a/src/browser/atom-window.coffee
+++ b/src/browser/atom-window.coffee
@@ -93,10 +93,9 @@ class AtomWindow
   hasProjectPath: -> @getLoadSettings().initialPaths?.length > 0
 
   setupContextMenu: ->
-    ContextMenu = null
+    ContextMenu = require './context-menu'
 
     @browserWindow.on 'context-menu', (menuTemplate) =>
-      ContextMenu ?= require './context-menu'
       new ContextMenu(menuTemplate, this)
 
   containsPaths: (paths) ->

